step.yml

```diff
@@ -298,10 +300,12 @@ inputs:
   - should_retry_test_on_fail: "no"
     opts:
       category: Debug
-      title: "Should retry test on failure?"
+      title: "Should retry tests on failure? (Not available in Xcode 13+)"
+      summary: If you set this input to `yes`, the Step will rerun your tests in case of failure.
```

Comment:
how about this: " If you set this input to 'yes', the Step will rerun your failed tests."